main/test/ca/uwaterloo/flix/util/FlixTest.scala

```diff
@@ -26,10 +26,7 @@ class FlixTest(name: String, path: String) extends FunSuite {
   /*
    * Include the standard library.
    */
-  val Library: List[String] = List(
```

Comment:
Right now Flix.scala includes all necessary libraries (but they could be moved here instead).